src/Holding/CreateHolding/CreateHolding.js

```diff
@@ -19,26 +19,28 @@ import {
 } from '../../common/hooks';
 import useCallout from '../../hooks/useCallout';
 import HoldingsForm from '../../edit/holdings/HoldingsForm';
+import { updateAffiliation } from '../../utils';
 
 const CreateHolding = ({
-  history,
   location,
   instanceId,
   referenceData,
   stripes,
   mutator,
+  tenantFrom,
 }) => {
   const callout = useCallout();
   const { instance, isLoading: isInstanceLoading } = useInstance(instanceId);
   const sourceId = referenceData.holdingsSourcesByName?.FOLIO?.id;
 
-  const onCancel = useCallback(() => {
-    history.push({
-      pathname: `/inventory/view/${instanceId}`,
-      search: location.search,
-    });
+  const goBack = useCallback(() => {
+    window.location.href = `/inventory/view/${instanceId}${location.search}`;
```

Comment:
Why do you use `window.location.href` instead of `react-router` API?

Reply:
@BogdanDenis To refresh the page, see the comment from Dmytro please, I described the process there